media/src/Poll.svelte

```diff
@@ -1,15 +1,27 @@
 <script>
-    import {forceNumber} from './utils';
+    //import {forceNumber} from './utils';
 
     export let currentPoll;
     export let socket;
 
     const handleOnSubmit = function(e) {
         e.preventDefault();
-
+        let response = null;
         const responseEl =
-              e.target.querySelector('input[name="poll_response"]');
-        const response = forceNumber(responseEl.value);
+            e.target.querySelectorAll('input[name="poll_response"]');
```

Comment:
I'm going to separate this code out as I think we should handle the different poll types differently here - I was just focusing on numerical input and I want to preserve the `forceNumber` cast here as I've found it's better to treat numbers as numbers rather then strings, in general.

Reply:
I refactored some things with the multiple-choice handling, too. Thanks for getting
this working.
I've found that using JS Array.forEach() makes clearer code than for loops in many cases:

```
             const responseEl =
                   e.target.querySelectorAll('input[name="poll_response"]');
             responseEl.forEach(function(el) {
                 if (el.checked) {
                     response.push(el.value);
                 }
             });
```